src/helpers/actions.ts

```diff
@@ -4,6 +4,15 @@ import { DEFAULT_NETWORK_ID, jsonParse, NETWORK_ID_WHITELIST } from './utils';
 
 export async function addOrUpdateSpace(space: string, settings: any) {
   if (!settings?.name) return false;
+  if (settings.domain) {
+    settings.domain = settings.domain?.replace(/(^\w+:|^)\/\//, '').replace(/\/$/, '');
+  }
+  if (settings.delegationPortal) {
+    settings.delegationPortal = {
+      ...settings.delegationPortal,
+      delegationNetwork: settings.delegationPortal.delegationNetwork ?? '1'
```

Comment:
Swapping these 2 lines would have allowed using just `delegationNetwork: '1'`

Reply:
you mean like this? 
```suggestion
      delegationNetwork: '1',
      ...settings.delegationPortal      
```

Reply:
https://github.com/snapshot-labs/snapshot-sequencer/pull/462